cmd/collectors/zapi/plugins/svm/svm.go

```diff
@@ -508,10 +515,17 @@ func (my *SVM) GetSSHData() (map[string]string, error) {
 		return nil, errs.New(errs.ErrNoInstance, "no records found")
 	}
 
+	compile, err := regexp.Compile("(.*)_cbc.*")
```

Comment:
better to move outside the function into a var and use `MustCompile` so we only pay the cost to compile the regex once

Reply:
Done